src/main/java/Listener/CountingEnforcement.java

```diff
@@ -39,25 +43,38 @@ private void countCheck(TextChannel channel, Message message, MessageAuthor auth
             return;
         }
 
-        Integer input;
-        User user;
-        if (channel.getMessages(2).get().getOldestMessage().get().getAuthor().getIdAsString().equals(BotInfo.getBotId())) {
-            input = Integer.parseInt(channel.getMessages(3).get().getOldestMessage().get().getContent().replaceAll("\\*", "")) + 1;
-            user = channel.getMessages(3).get().getOldestMessage().get().getAuthor().asUser().get();
+        Integer previousNum;
+        User previousUser;
+        MessageSet lastTwoMessages = channel.getMessages(2).get();
```

Comment:
I tried to minimize the number of redundant API calls here

Reply:
see https://github.com/IKanraI/Gaiza/pull/138#discussion_r661870610 for more potential improvement